chillFlix/src/app/components/movie-list/movie-list.component.html

```diff
@@ -1,16 +1,17 @@
 <!--component html goes here -->
-<div style="text-align:center">
+<div style="text-align:center" >
```

Comment:
er word veel gecentreerd, dit zou of in een parent component moeten (alles centreren), of een standaard layout voor gemaakt moeten worden